components/nexus-repository/src/main/java/org/sonatype/nexus/repository/search/SearchServiceImpl.java

```diff
@@ -100,7 +100,8 @@ public void createIndex(final Repository repository) {
     checkNotNull(repository);
     // TODO we should calculate the checksum of index settings and compare it with a value stored in index _meta tags
     // in case that they not match (settings changed) we should drop the index, recreate it and re-index all components
-    if (!client.get().admin().indices().prepareExists(repository.getName()).execute().actionGet().isExists()) {
+    if (!client.get().admin().indices().prepareExists(Strings2.lower(repository.getName())).execute().actionGet()
```

Comment:
Might be safer to reuse the existing `safeIndexName(repository)` method for consistency?